### DISCUSSION TITLE
I ran Lailloken UI.ahk file and nothing happened after

### DISCUSSION THREAD

**Discussion by dchoi44** [Lailloken/Exile-UI#53] [Q&A]
And it is terminated by itself after some time. 
Am I missing something?

**Comment by Lailloken** [comment]
Which PoE client and resolution are you running?
EDIT: I just saw on your profile that you're using the Korean client. I made some changes to the file on main, so you could download the modified file or copy the changes to try it out:
`GroupAdd, poe_window, ahk_class POEWindowClass` (add this to the block around line 30)

**Comment by Lailloken** [comment]
I have added potential support for the Korean client to v1.24.4, you can test it if you want to.

**Comment by UMSpectre** [comment]
![Screenshot 2023-04-08 120209](https://user-images.githubusercontent.com/130247998/230738758-083db219-7558-4bb3-8910-6595b4c3aaa9.png)
This error pops up when I try to run the same file.

**Reply by Lailloken** [reply]
Some people fixed it by creating a desktop shortcut of the file

**Reply by UMSpectre** [reply]
I attempted to use a desktop shortcut but it still gave me the same error unfortunately.

**Reply by Lailloken** [reply]
Someone also mentioned they moved the script folder into the game folder and then it worked.
I'm pretty sure it's an AutoHotkey or file-access problem, so you could also try updating/reinstalling or running as admin.

**Reply by UMSpectre** [reply]
Well that isn't working either. No matter what I do I get the same error. Seems like I wont be able to use this or exile leveling since that wont work too. I really wanted to use them in conjunction with each other.

**Reply by Lailloken** [reply]
This is another solution found in a thread, starting the script via powershell:
`Start-Process -FilePath "X:\Lailloken-UI-1.29.7\Lailloken UI.ahk" -WorkingDirectory "X:\Lailloken-UI-1.29.7"`
replace the folders with your folder-structure and maybe it'll work.

**Comment by UMSpectre** [comment]
I attempted that and it still gives me the same error message. : (

**Reply by Lailloken** [reply]
I have added an experimental Launcher.ahk file into the build on main. You can download it via the download button on the front-page. If it doesn't work initially, please launch it while holding down the shift-key. A text-box will open and show the working directory. I need to know what it says because I'm suspecting AutoHotkey not reading the directory correctly for some reason.

**Comment by UMSpectre** [comment]
Ok, is this what's supposed to happen?
![Script error message](https://user-images.githubusercontent.com/130247998/230757529-4c7a3c16-49c4-426c-8224-52a047e84644.png)

**Reply by Lailloken** [reply]
Alright, so the issue is not that AHK cannot read the current folder.
I assume the CustomFont error still appears, even when using the launcher.ahk file? Then it's more likely that AHK cannot access the folder for some reason. I personally would now uninstall AHK, reboot, delete the AHK install folder, reinstall, reboot, then test again. It's up to you if you want to test that as well.

In the meantime, I'll try to replicate this issue on my end because quite a few people have reported it in the last few days and because there were so many ways they reportedly fixed it.
EDIT: Is OneDrive sync on while you're trying to run the script? Maybe that's stopping it from running correctly.

**Reply by Lailloken** [reply]
I have added another potential fix, please redownload the script via the download button. This should make it possible to simply launch Lailloken UI.ahk

**Reply by UMSpectre** [reply]
So, when I tried to run the script it created an empty ini folder and that's it. I don't know anything about code but I looked in the LUI script. It showed gdiplus failed. I looked it up, found a file repair program. Now when I launched it, it put an icon in the hidden taskbar menu. I opened th

at and that image is attached. Also, it only stays open for about a minute. I'm not sure if that's what should happen. So, I 'm not sure if I did something or what.
![LUI](https://user-images.githubusercontent.com/130247998/230790532-4a2c1463-9f57-451f-8e74-d01aa6a5ac01.png)

**Reply by Lailloken** [reply]
In that minute, it's waiting for the game to be launched / for a client-window to be present. It's a good sign if it gets that far without throwing error messages at you.

**Reply by UMSpectre** [reply]
Finally, it's working now. I appreciate the help and quick responses.
![Screenshot 2023-04-09 114829](https://user-images.githubusercontent.com/130247998/230791178-b9563557-db25-44d0-a576-198197fec3f6.png)

**Reply by Lailloken** [reply]
Really glad it worked out, glhf
